src/components/ChildHideCheckbox.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from 'react';
+import { Checkbox, Tooltip } from 'antd';
+
+import { ChildCheckboxProps } from '../types';
+import { VisibilityContext } from '../AgentVisibilityContext';
+import { CheckboxState, tooltipMap } from '../constants';
+
+const ChildHideCheckbox: React.FunctionComponent<ChildCheckboxProps> = (
+  props: ChildCheckboxProps
+): JSX.Element => {
+  const { name, parentName } = props;
+  const { handleHideChildCheckboxChange, hiddenAgents } =
+    useContext(VisibilityContext);
+
+  const selections = hiddenAgents[parentName];
+
+  const getCheckboxStatus = () => {
+    if (selections?.includes(name)) {
+      return CheckboxState.Unchecked;
+    }
+    return CheckboxState.Checked;
+  };
+
+  const checkboxStatus = getCheckboxStatus();
+  const tooltipText = tooltipMap[checkboxStatus];
+
+  return (
+    <Tooltip placement="right" title={tooltipText}>
```

Comment:
```suggestion
    <Tooltip placement="right" title={tooltipText} trigger=["focus", "hover"]>
```
For this and other tooltips, consider making them responsive to keyboard focusing. It might also be nice to add a visually hidden label?

Reply:
Will add this change! Focus states inside jupyer notebooks is a topic I'm scared to open at the moment but this is a good change for now.